packages/calling-stateful-client/src/TestUtils.ts

```diff
@@ -347,7 +347,7 @@ export const createMockLocalVideoStream = (): LocalVideoStream =>
     switchSource: Promise.resolve,
     /* @conditional-compile-remove(video-background-effects) */
     feature: () => createMockVideoEffectsAPI()
-  } as unknown as LocalVideoStream);
+  }) as unknown as LocalVideoStream;
```

Comment:
Just wanted to confirm that the meaning of the expression stays the same

Reply:
It's a change in prettier / eslint syntax